public/app/plugins/datasource/prometheus/types.ts

```diff
@@ -32,6 +32,8 @@ export interface PromOptions extends DataSourceJsonData {
   prometheusType?: PromApplication;
   prometheusVersion?: string;
   defaultEditor?: QueryEditorMode;
+  incrementalQuerying?: boolean;
+  incrementalQueryOverlapDuration?: number; // in seconds
```

Comment:
why do we call it `duration` in some places and `window` in others? (i prefer `window`, personally)

Reply:
I screwed up the refactor 🤦